resources/views/event/gallery/index.blade.php

```diff
@@ -19,6 +19,11 @@
                         <a class="example-image-link"  href="{{asset($path.$photo->name)}}" data-lightbox="roadtrip" data-title="{{$photo->caption}}">
                             <img class="example-image" src="{{asset($path.$photo->name)}}" style="max-width:100%;">
                         </a>
+                        @if($creator)
+                            {!! Form::model(null, array( 'method' => 'POST','action' =>array('Event\EventGalleryController@destroy',$photo->id))) !!}
+                            {!! Form::submit('delete photo', array( 'class'=>'btn btn-danger form-control','onclick'=>"return confirm('Are you sure?');")) !!}
+                            {!! Form::close() !!} 
+                        @endif
```

Comment:
it will be something of the following form
`<form action="" method="POST">
                      {!! csrf_field() !!}
                      {!! method_field('DELETE') !!}
                      <button type="submit" onclick="return confirm('Are you sure?');" class="btn btn-danger btn-event">Delete</button>
                 </form>`